python/tvm/relay/op/nn/nn.py

```diff
@@ -86,3 +89,30 @@ def conv2d(data,
     return _make.conv2d(data, weight, strides, padding, dilation,
                         groups, channels, kernel_size, data_layout,
                         weight_layout, out_layout, out_dtype)
+
+
+def pad(data,
+        pad_width,
+        pad_value=None):
+    r"""Padding
+
+    This operator takes in a tensor and pads each axis by the specified
+    widths using the specified value.
+
+    Parameters
+    ----------
+    data: relay.Expr
+        The input data to the operator
+    pad_width: tuple of <tuple of <int>>, required
+        Number of values padded to the edges of each axis, in the format
+        of ((before_1, after_1), ..., (before_N, after_N))
+    pad_value: relay.Expr, optional, default=0.0
+        The value used for padding
+
+    Returns
+    -------
+    result : relay.Expr
+        The computed result.
+    """
+    return _make.pad(data, pad_width,
+                     pad_value if pad_value is not None else relay.Constant(tvm.nd.array(0)))
```

Comment:
This was very ugly, so my question is, is there a better way? Is there a better place I can put the default value of this (Relay) argument? It does not affect the type so it should not be an attr (at least I don't think).

Reply:
Let us make pad_value as a double attribute for now. Although putting it as relay.Expr is more general, most usecases don;t need it